flink114/manager/src/main/scala/pl/touk/nussknacker/engine/management/CustomFlinkStreamingDeploymentManagerProvider.scala

```diff
@@ -0,0 +1,7 @@
+package pl.touk.nussknacker.engine.management
+
+import pl.touk.nussknacker.engine.management.common.CommonFlinkStreamingDeploymentManagerProvider
```

Comment:
Why do we need it at all?

Reply:
We need to return custom StreamingRestManager that differently handles checking if job is running. I'm not sure why it is necessary, but it was there before and timeouts happen without it.